backend/src/main/java/com/tourneynizer/tourneynizer/dao/TeamRequestDao.java

```diff
@@ -104,12 +126,29 @@ public TeamRequest findById(long requestId) {
         }
     }
 
-    public void removeRequest(TeamRequest teamRequest) {
+    public int removeRequest(TeamRequest teamRequest) {
         String sql = "DELETE FROM teamRequest WHERE id=?";
-        this.jdbcTemplate.update(connection -> {
+        return this.jdbcTemplate.update(connection -> {
+            PreparedStatement preparedStatement = connection.prepareStatement(sql);
+            preparedStatement.setLong(1, teamRequest.getId());
+            return preparedStatement;
+        });
+    }
+
+    public int declineRequest(TeamRequest teamRequest) throws IllegalArgumentException {
+        if (teamRequest.isAccepted() != null) {
+            String happened = teamRequest.isAccepted() ? "accepted" : "declined";
+            throw new IllegalArgumentException("That request has already been " + happened);
+        }
+
+        String sql = "UPDATE teamRequest SET accepted=False WHERE id=?";
+        int updated = this.jdbcTemplate.update(connection -> {
             PreparedStatement preparedStatement = connection.prepareStatement(sql);
             preparedStatement.setLong(1, teamRequest.getId());
             return preparedStatement;
         });
+
+        teamRequest.setAccepted(false);
+        return updated;
     }
```

Comment:
Are you still gonna be returning declined requests to the front end? Wouldn't declined requests be something that we wouldn't want to display so we should make declined requests hidden from the front end.

Reply:
Not if the user wants to see that their request got declined.